<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="WS-STA146" name="wssim.CustomerUnitTest" tests="5" failures="0" timestamp="6 Mar 2017 19:32:06 GMT" time="0.932" errors="5">
  <testcase name="oneTonOneInsertBeforeDeleting" time="0.001" classname="wssim.CustomerUnitTest">
    <error type="java.lang.NoClassDefFoundError" message="Could not initialize class io.ebean.Ebean">
      <![CDATA[java.lang.NoClassDefFoundError: Could not initialize class io.ebean.Ebean
	at io.ebean.Model.db(Model.java:116)
	at io.ebean.Model.save(Model.java:190)
	at wssim.CustomerUnitTest.oneTonOneInsertBeforeDeleting(CustomerUnitTest.java:39)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
    </error>
  </testcase> <!-- oneTonOneInsertBeforeDeleting -->
  <testcase name="deleteOneToManySetNull" time="0.928" classname="wssim.CustomerUnitTest">
    <error type="java.lang.ExceptionInInitializerError">
      <![CDATA[java.lang.ExceptionInInitializerError
	at io.ebean.Model.db(Model.java:116)
	at io.ebean.Model.save(Model.java:190)
	at wssim.CustomerUnitTest.deleteOneToManySetNull(CustomerUnitTest.java:145)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
Caused by: java.lang.RuntimeException: javax.persistence.PersistenceException: Failed to run script
	at io.ebean.Ebean$ServerManager.<init>(Ebean.java:164)
	at io.ebean.Ebean$ServerManager.<init>(Ebean.java:124)
	at io.ebean.Ebean.<clinit>(Ebean.java:118)
	... 27 more
Caused by: javax.persistence.PersistenceException: Failed to run script
	at io.ebean.dbmigration.DdlGenerator.runScript(DdlGenerator.java:119)
	at io.ebean.dbmigration.DdlGenerator.runCreateSql(DdlGenerator.java:145)
	at io.ebean.dbmigration.DdlGenerator.runDdl(DdlGenerator.java:88)
	at io.ebean.dbmigration.DdlGenerator.execute(DdlGenerator.java:63)
	at io.ebeaninternal.server.core.DefaultServer.executePlugins(DefaultServer.java:289)
	at io.ebeaninternal.server.core.DefaultContainer.createServer(DefaultContainer.java:134)
	at io.ebeaninternal.server.core.DefaultContainer.createServer(DefaultContainer.java:77)
	at io.ebeaninternal.server.core.DefaultContainer.createServer(DefaultContainer.java:44)
	at io.ebean.EbeanServerFactory.create(EbeanServerFactory.java:55)
	at io.ebean.Ebean$ServerManager.getWithCreate(Ebean.java:201)
	at io.ebean.Ebean$ServerManager.<init>(Ebean.java:159)
	... 29 more
Caused by: java.sql.SQLException: Error executing stmt[create or replace view order_agg_vw as select d.order_id as id, d.order_id as order_id, sum(d.order_qty * d.unit_price) as order_total, sum(d.ship_qty * d.unit_price) as ship_total, sum(d.ship_qty * d.unit_price) as ship_total2 from o_order_detail d group by d.order_id] error[Table "O_ORDER_DETAIL" not found; SQL statement:
create or replace view order_agg_vw as select d.order_id as id, d.order_id as order_id, sum(d.order_qty * d.unit_price) as order_total, sum(d.ship_qty * d.unit_price) as ship_total, sum(d.ship_qty * d.unit_price) as ship_total2 from o_order_detail d group by d.order_id [42102-191]]
	at io.ebean.dbmigration.ddl.DdlRunner.runStatement(DdlRunner.java:94)
	at io.ebean.dbmigration.ddl.DdlRunner.runStatements(DdlRunner.java:60)
	at io.ebean.dbmigration.ddl.DdlRunner.runAll(DdlRunner.java:40)
	at io.ebean.dbmigration.DdlGenerator.runScript(DdlGenerator.java:111)
	... 39 more
Caused by: org.h2.jdbc.JdbcSQLException: Table "O_ORDER_DETAIL" not found; SQL statement:
create or replace view order_agg_vw as select d.order_id as id, d.order_id as order_id, sum(d.order_qty * d.unit_price) as order_total, sum(d.ship_qty * d.unit_price) as ship_total, sum(d.ship_qty * d.unit_price) as ship_total2 from o_order_detail d group by d.order_id [42102-191]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.command.Parser.readTableOrView(Parser.java:5349)
	at org.h2.command.Parser.readTableFilter(Parser.java:1245)
	at org.h2.command.Parser.parseSelectSimpleFromPart(Parser.java:1884)
	at org.h2.command.Parser.parseSelectSimple(Parser.java:2032)
	at org.h2.command.Parser.parseSelectSub(Parser.java:1878)
	at org.h2.command.Parser.parseSelectUnion(Parser.java:1699)
	at org.h2.command.Parser.parseSelect(Parser.java:1687)
	at org.h2.command.Parser.parseCreateView(Parser.java:4785)
	at org.h2.command.Parser.parseCreate(Parser.java:4160)
	at org.h2.command.Parser.parsePrepared(Parser.java:360)
	at org.h2.command.Parser.parse(Parser.java:315)
	at org.h2.command.Parser.parse(Parser.java:287)
	at org.h2.command.Parser.prepareCommand(Parser.java:252)
	at org.h2.engine.Session.prepareLocal(Session.java:560)
	at org.h2.engine.Session.prepareCommand(Session.java:501)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1188)
	at org.h2.jdbc.JdbcPreparedStatement.<init>(JdbcPreparedStatement.java:73)
	at org.h2.jdbc.JdbcConnection.prepareStatement(JdbcConnection.java:276)
	at org.avaje.datasource.pool.PooledConnection.prepareStatement(PooledConnection.java:401)
	at org.avaje.datasource.pool.PooledConnection.prepareStatement(PooledConnection.java:374)
	at io.ebean.dbmigration.ddl.DdlRunner.runStatement(DdlRunner.java:86)
	... 42 more
]]>
    </error>
  </testcase> <!-- deleteOneToManySetNull -->
  <testcase name="oneToManySetDeleteWithNull" time="0.001" classname="wssim.CustomerUnitTest">
    <error type="java.lang.NoClassDefFoundError" message="Could not initialize class io.ebean.Ebean">
      <![CDATA[java.lang.NoClassDefFoundError: Could not initialize class io.ebean.Ebean
	at io.ebean.Model.db(Model.java:116)
	at io.ebean.Model.save(Model.java:190)
	at wssim.CustomerUnitTest.oneToManySetDeleteWithNull(CustomerUnitTest.java:122)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
    </error>
  </testcase> <!-- oneToManySetDeleteWithNull -->
  <testcase name="oneTonOneNotDeleteNullEntity_Customer" time="0.001" classname="wssim.CustomerUnitTest">
    <error type="java.lang.NoClassDefFoundError" message="Could not initialize class io.ebean.Ebean">
      <![CDATA[java.lang.NoClassDefFoundError: Could not initialize class io.ebean.Ebean
	at io.ebean.Model.db(Model.java:116)
	at io.ebean.Model.save(Model.java:190)
	at wssim.CustomerUnitTest.oneTonOneNotDeleteNullEntity_Customer(CustomerUnitTest.java:96)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
    </error>
  </testcase> <!-- oneTonOneNotDeleteNullEntity_Customer -->
  <testcase name="oneTonOneNotDeleteNullEntity_Address" time="0.001" classname="wssim.CustomerUnitTest">
    <error type="java.lang.NoClassDefFoundError" message="Could not initialize class io.ebean.Ebean">
      <![CDATA[java.lang.NoClassDefFoundError: Could not initialize class io.ebean.Ebean
	at io.ebean.Model.db(Model.java:116)
	at io.ebean.Model.save(Model.java:190)
	at wssim.CustomerUnitTest.oneTonOneNotDeleteNullEntity_Address(CustomerUnitTest.java:68)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
    </error>
  </testcase> <!-- oneTonOneNotDeleteNullEntity_Address -->
</testsuite> <!-- wssim.CustomerUnitTest -->
